### ISSUE TITLE
Upper extremity claudication

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11449]
**Preferred term label:**

**Synonyms**
Limb claudication

**Definition (free text, please give PubMed ID)**

PMID: 36344353 Vasculitis of the aorta and its branches can also cause vascular symptoms somehow resembling those more frequently reported by patients with Takayasu arteritis. The most typical are arteriodynia, a burning pain over the area of an inflamed artery (e.g., carotidynia), and **limb claudication**, an ischemic pain caused by acute or chronic stenosis of a limb artery.

PMID: 37021621 Patients can develop symptoms of **upper extremity claudication** because of axillary or subclavian artery disease.

PMID: 30385998 Temporal artery tenderness or bulging of the artery may occur and on physical exam one may find the artery to be nodular or indurated. Aortic arch syndrome is not common nor is lower or **upper extremity claudication** but may be present due to involvement of the large arteries such as the femoral or brachial arteries. 

**Parent term (use hpo.jax.org/app)**
Abnormal arterial physiology HP:0025323

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell arteritis ORPHA:397

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001462
- label: Upper extremity claudication
- parents: [Intermittent claudication (HP:0004417)](https://hpo.jax.org/app/browse/term/HP:0004417)
- pmid: PMID:21250079|PMID:37021621
- def: Pain in an arm that is triggered by muscular exertion.
- comment: This anomaly is observed in the setting of a patient with axillary or subclavian artery disease.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11449
- synonyms:
